integration_tests/general.bats

```diff
@@ -34,6 +34,16 @@ teardown_file() {
    $IPROF -r
 }
 
+function run_mpi_helloworld_with_no_analysis() {
+   mpicc ./integration_tests/mpi_helloworld.c -o mpi_helloworld
+   THAPI_SYNC_DAEMON=fs timeout 40s $MPIRUN -n 2 $IPROF --no-analysis -- ./mpi_helloworld 0 2>/dev/null
+}
+
+@test "no-analysis_output" {
+   run_output=$(run_mpi_helloworld_with_no_analysis)
+   [[ "${run_output}" == "" ]]
```

Comment:
We can do  `! grep -v Hello world from process | Trace Location`  (aka we should not find anything else ecept "Hello world from process" and trace location